ingestion/setup.py

```diff
@@ -57,6 +57,7 @@
     "giturlparse": "giturlparse",
     "validators": "validators~=0.22.0",
     "teradata": "teradatasqlalchemy>=20.0.0.0",
+    "cockroach": "sqlalchemy-cockroachdb==2.0.2",
```

Comment:
maybe we can be more lax here and stick to ~2?

Reply:
Did the required change